src/Cardano/Prelude/GHC/Heap/NormalForm/Classy.hs

```diff
@@ -510,6 +511,15 @@ deriving via UseIsNormalForm ZonedTime        instance NoUnexpectedThunks ZonedT
 -- thunks.
 deriving via OnlyCheckIsWHNF "Strict.ByteString" BS.Strict.ByteString instance NoUnexpectedThunks BS.Strict.ByteString
 
+-- | A short bytestring is defined as:
+--
+-- > data ByteString = PS {-# UNPACK #-} !(ForeignPtr Word8) -- payload
+-- >                      {-# UNPACK #-} !Int                -- offset
+-- >                      {-# UNPACK #-} !Int                -- length
```

Comment:
That's a regular strict `ByteString`, for a `ShortByteString` see https://hackage.haskell.org/package/bytestring-0.10.10.0/docs/src/Data.ByteString.Short.Internal.html#ShortByteString

Reply:
:man_facepalming: Thanks. I'll change that.